<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgGestionObras</class>
 <widget class="QWidget" name="dlgGestionObras">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1031</width>
    <height>732</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="tvObras">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>951</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QRadioButton" name="rbTodos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Todo</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbManuscritos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Manuscritos</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbImpresos">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>99</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Impresos</string>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="ckSinMateria">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>84</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sin materia</string>
   </property>
  </widget>
  <widget class="QListWidget" name="lwTemas">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>256</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAnadirTema">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>AÃ±adir tema</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbQuitarTema">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitar tema</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbQuitarTemasTodos">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>260</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Quitar todos</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
